sys/net/gnrc/pktbuf/gnrc_pktbuf.c

```diff
@@ -109,5 +109,32 @@ gnrc_pktsnip_t *gnrc_pktbuf_reverse_snips(gnrc_pktsnip_t *pkt)
     return reversed;
 }
 
+int gnrc_pktbuf_merge(gnrc_pktsnip_t *pkt)
```

Comment:
overall, the implementation makes sense!

Reply:
Great!